docs/getting-started.md

```diff
@@ -0,0 +1,344 @@
+
+> IMPORTANT: If you don't specify a release name, one will be automatically assigned.
+
+To install the chart, specify any release name of your choice. For example, to install OSDFIR Infrastructure using a release name of `my-release`, run:
+
+```shell
+$ helm install my-release oci://us-docker.pkg.dev/osdfir-registry/osdfir-charts/osdfir-infrastructure
+```
+
+> NOTE: Check the configurable parameters of the OSDFIR Infrastructure chart and their default values at the [GitHub repository](https://github.com/google/osdfir-infrastructure/tree/main/charts/osdfir-infrastructure#parameters).
+
+Once you have the chart installed a "Notes" section will be shown at the bottom of the installation information. It contains important instructions about how to access Turbinia and Timesketch and retrieve credentials. Please check it carefully:
+
+![notes-txt](images/notes-txt.png)
+
+> IMPORTANT: When deploying on Minikube, you may see errors such as *CrashLoopBackOff* and your application may fail to start. This is typically because the Persistent Volumes are still provisioning or the docker images are still being pulled and may need to wait a few minutes for the error to resolve.
+
+Find how to install Turbinia or Timesketch in the examples below:
+
+* To install Turbinia on Minikube with a release name of `my-release`, run:
+
+    ```shell
+    $ helm install my-release oci://us-docker.pkg.dev/osdfir-registry/osdfir-charts/turbinia
+    ```
+
+    > NOTE: Check the configurable parameters of the Turbinia chart and their default values at the [GitHub repository](https://github.com/google/osdfir-infrastructure/tree/main/charts/turbinia#parameters).
+
+* To install Timesketch on Minikube with a release name of `my-release`, run:
+
+    ```shell
+    $ helm install my-release oci://us-docker.pkg.dev/osdfir-registry/osdfir-charts/timesketch
+    ```
+
+    > NOTE: Check the configurable parameters of the Timesketch chart and their default values at the [GitHub repository](https://github.com/google/osdfir-infrastructure/tree/main/charts/timesketch#parameters).
+
+Now, you can manage your deployments from the Kubernetes Dashboard. Follow the instructions below to access the Web user interface.
+
+### Step 6: Access the Kubernetes Dashboard
+
+The [Kubernetes Dashboard](https://github.com/kubernetes/dashboard) is a Web user interface from which you can manage your clusters in a more simple and digestible way. It provides information on the cluster state, deployments and container resources. You can also check both the credentials and the log error file of each pod within the deployment.
+
+To open the Kubernetes Dashboard, you just need to run the following command:
+
+```shell
+$ minikube dashboard
```

Comment:
to prevent issues with people doing this over a remote ssh connection, I would go for `minikube dashboard --url` instead of letting minikube try to open a browser

Reply:
Updated